packages/nx/src/plugins/project-json/build-nodes/project-json.ts

```diff
@@ -4,15 +4,16 @@ import { ProjectConfiguration } from '../../../config/workspace-json-project-jso
 import { toProjectName } from '../../../config/workspaces';
 import { readJsonFile } from '../../../utils/fileutils';
 import { NxPluginV2 } from '../../../project-graph/plugins';
+import { CreateNodesError } from '../../../project-graph/error-types';
 
 export const ProjectJsonProjectsPlugin: NxPluginV2 = {
   name: 'nx/core/project-json',
   createNodes: [
     '{project.json,**/project.json}',
     (file, _, { workspaceRoot }) => {
-      const json = readJsonFile<ProjectConfiguration>(
-        join(workspaceRoot, file)
-      );
+      let json: ProjectConfiguration;
+      json = readJsonFile<ProjectConfiguration>(join(workspaceRoot, file));
+
```

Comment:
These changes are not necessary?

Reply:
yep those were leftover from trying stuff out